### ISSUE TITLE
learning rate and steps to train synthesizer in new data

### ISSUE THREAD

**Issue by majidAdibian77** [CorentinJ/Real-Time-Voice-Cloning#1038]
Hi
I found following parameters in training synthesizer. First question is that this parameters are for which data? 
```
tts_schedule = [(2,  1e-3,  20_000,  12),   # Progressive training schedule
                (2,  5e-4,  40_000,  12),   # (r, lr, step, batch_size)
                (2,  2e-4,  80_000,  12),   #
                (2,  1e-4, 160_000,  12),   # r = reduction factor (# of mel frames
                (2,  3e-5, 320_000,  12),   #     synthesized for each decoder iteration)
                (2,  1e-5, 640_000,  12)],  # lr = learning rate
```
In my case I have about 92000 utterances from 60 speaker in new language so second question is that what lr and steps may be best using batch-size=32 ? Or generally how to change these parameters for new data?
